README.md

```diff
@@ -84,17 +90,19 @@ manually build your preferred version and use it with BitShares by specifying it
   Example: `cmake -DOPENSSL_ROOT_DIR=/path/to/openssl ..`
 
 
-### Run the node software
+### Run
+
+**Run Node Software:**
 
-**After Building**, the node software `witness_node` can be launched with:
+Stay on `bitshares-core` directory before you run the below `witness_node` command
 
     ./programs/witness_node/witness_node
 
-The node will automatically create a `witness_node_data_dir` directory with some config files.
-The blockchain data will be stored in the directory too.
-It may take several hours to fully synchronize the blockchain.
+The node run will automatically create the `witness_node_data_dir` sub-directory along with node config files then start synchronizing with blockchain.
```

Comment:
I think `start synchronizing the blockchain` is better. If to add `with`, maybe write `with the network` or so.